### PR TITLE
WIP: add content to sample data

### PR DESCRIPTION
**Pull request by ninsbl**
This is work in progress. I have to add links and some description and do some cleaning.
Feedback on the general direction of the way content is presented is welcome already now.

### CODE REVIEW

File: content/download/data.en.md [hunk 1]
```diff
@@ -1,39 +1,89 @@
...
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" class="inl btn btn-primary" target="_blank">Download ZIP  (50MB)</a>
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.tar.gz" class="inl btn btn-secondary">Download TAR.GZ  (50MB)</a>
+  </li>
+</ul>
+A smaller subset of the North Carolina location:
+<hr>
+
+
+### <a name="SpearFishDataset"></a> Spearfish dataset
+<hr>
```
Comment (lucadelu) [inline content/download/data.en.md]: Spearfish dataset is missing

File: content/download/data.en.md [hunk 2]
```diff
@@ -1,39 +1,89 @@
...
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip" class="inl btn btn-primary" target="_blank">Download ZIP (159MB)</a>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz" class="inl btn btn-secondary">Download TAR.GZ (159MB)</a>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" target="_blank"> nc_basic_spm_grass7 </a></span>
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" class="inl btn btn-primary" target="_blank">Download ZIP  (50MB)</a>
+  <a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.tar.gz" class="inl btn btn-secondary">Download TAR.GZ  (50MB)</a>
+  </li>
+</ul>
+A smaller subset of the North Carolina location:
```
Comment (lucadelu) [inline content/download/data.en.md]: this sentence is not clear to me

File: content/download/data.en.md [hunk 3]
```diff
@@ -1,39 +1,89 @@
...
+
+### <a name="NorthCarolinaDataset"></a> North Carolina data set
+
+This dataset is a modern package of geospatial data from North Carolina, USA. It offers raster, vector, LiDAR and satellite data.
+Most of the examples in the module manuals are written with this dataset. Furthermore, the tessuite operates with this dataset.
+The description, list of maps, and a quick usage tutorial are available here.
+The North Carolina dataset can be downloaded in two versions:
+<ul id="links" class="list-unstyled version">
+ <li>
+  <span class="mwl"><a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip " target="_blank"> nc_spm_full_v2alpha </a></span>
```
Comment (lucadelu) [inline content/download/data.en.md]: I would rename it to Full dataset or something similar

File: content/download/data.en.md [hunk 4]
```diff
@@ -1,39 +1,89 @@
...
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha.zip" class="inl btn btn-primary" target="_blank">Download ZIP (159MB)</a>
+  <a href="http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz" class="inl btn btn-secondary">Download TAR.GZ (159MB)</a>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip" target="_blank"> nc_basic_spm_grass7 </a></span>
```
Comment (lucadelu) [inline content/download/data.en.md]: I would rename it the Simple (or Basic) dataset

File: content/download/data.en.md (hunk 3 above)
Comment (veroandreo) [reply content/download/data.en.md]: Is that alpha version the same than the current one here: https://grass.osgeo.org/sampledata/north_carolina/nc_spm_08_grass7.zip? I would just use the one from the old website (which is the one we can download from the startup) and name it "North Carolina full dataset"

File: content/download/data.en.md (hunk 4 above)
Comment (veroandreo) [reply content/download/data.en.md]: Yes, to keep it consistent "North Carolina basic dataset"

File: content/download/data.en.md (hunk 2 above)
Comment (veroandreo) [reply content/download/data.en.md]: I guess it must go before the basic dataset links, no?

File: content/download/data.en.md [hunk 5]
```diff
@@ -1,39 +1,89 @@
...
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> LiDAR </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">LAS (21MB)</a> <a href="#" class="inl btn btn-secondary">LAZ (6MB)</a>
+  <p>extra Raleigh LiDAR dataset (note: height in feet):  multi-return</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Climate </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (690MB)</a>
+  <p>GRASS 7 Climatic data time series NC location nc_climate_spm_2000_2012 - related tutorial Separate NC climatic data location</p>
```
Comment (veroandreo) [inline content/download/data.en.md]: link missing for the tutorial maybe?

File: content/download/data.en.md [hunk 6]
```diff
@@ -1,39 +1,89 @@
...
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Time series </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
+  <p>Extra time series of MODIS Land Surface Temperature (mapset to unzip in NC location): MODIS time series</p>
```
Comment (veroandreo) [inline content/download/data.en.md]:
```suggestion
  <p>Extra time series of MODIS Land Surface Temperature (mapset to unzip in NC location): MODIS LST time series</p>

```

File: content/download/data.en.md [hunk 7]
```diff
@@ -1,39 +1,89 @@
...
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Projection </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
+  <p>NAD83(HARN) / North Carolina, EPSG 3358)</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> 3D / voxel </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (8.9MB)</a>
+  <p>complete Slovakia 3D precipitation location: Slovakia 3D precipitation voxel data set</p>
```
Comment (veroandreo) [inline content/download/data.en.md]:
```suggestion
  <p>complete Slovakia 3D precipitation location: Slovakia 3D precipitation voxel dataset</p>

```

File: content/download/data.en.md (hunk 5 above)
Comment (ninsbl) [reply content/download/data.en.md]: Yes, I have not yet managed to get to those details.

File: content/download/data.en.md [hunk 8]
```diff
@@ -1,39 +1,89 @@
...
+<i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the download button on the GRASS GIS startup screen.
+</div>
+
+<hr>
+
+### <a name="NorthCarolinaDataset"></a> North Carolina data set
+
+This dataset is a modern package of geospatial data from North Carolina, USA. It offers raster, vector, LiDAR and satellite data.
+Most of the examples in the module manuals are written with this dataset. Furthermore, the tessuite operates with this dataset.
+The description, list of maps, and a quick usage tutorial are available here.
```
[the remark this answers is not in the document]
Comment (ninsbl) [reply content/download/data.en.md]: Yes, in progress.

File: content/download/data.en.md [hunk 9]
```diff
@@ -1,39 +1,89 @@
...
+<i class="fa fa-arrow-right"></i> This section gathers sample datasets available as ready-to-use GRASS GIS <b>locations</b> and other common GIS formats.
+</div>
+
+<div class="alert rounded-0 alert-default">
+<i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the download button on the GRASS GIS startup screen.
+</div>
+
+<hr>
+
+### <a name="NorthCarolinaDataset"></a> North Carolina data set
```
[the remark this answers is not in the document]
Comment (ninsbl) [reply content/download/data.en.md]: There are plenty of cases with "dataset", though "data set" is correct as well. We should try to have it consistent. Should i use "data set" throughout the page?

File: content/download/data.en.md [hunk 10]
```diff
@@ -1,39 +1,107 @@
...
+<hr>
+
+
+### <a name="SpecialTopicDatasets"></a> Datasets for special topics
+<hr>
+
+<ul id="links" class="list-unstyled version">
+ <li>
+  <span class="mwl"><a href="http://grassbook.org/datasets/datasets-3rd-edition/" target="_blank"> Data import </a></span>
+  <a href="http://grassbook.org/datasets/datasets-3rd-edition/" class="inl btn btn-primary" target="_blank">Download ZIP</a>
```
Comment (neteler) [inline content/download/data.en.md]:
```suggestion
  <span class="mwl"><a href="https://grassbook.org/datasets/datasets-3rd-edition/" target="_blank"> Data import </a></span>
  <a href="https://grassbook.org/datasets/datasets-3rd-edition/" class="inl btn btn-primary" target="_blank">Download ZIP</a>

```

File: content/download/data.en.md [hunk 11]
```diff
@@ -1,39 +1,107 @@
...
+  <p>Selected maps in common GIS formats (SHAPE, KML, GeoTIFF) including RGB orthophoto</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> LiDAR </a></span>
```
Comment (neteler) [inline content/download/data.en.md]:
```suggestion
  <span class="mwl"><a href="https://grass.osgeo.org/grass-stable/manuals/" target="_blank"> LiDAR </a></span>
```

(we now have a redirect)

File: content/download/data.en.md [hunk 12]
```diff
@@ -1,39 +1,107 @@
...
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Time series </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
+  <p>Extra time series of MODIS Land Surface Temperature (mapset to unzip in NC location): MODIS LST time series</p>
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/grass78/manuals/index.html " target="_blank"> Projection </a></span>
+  <a href="https://grass.osgeo.org/grass78/manuals/index.html" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
+  <p>NAD83(HARN) / North Carolina, EPSG 3358)</p>
```
Comment (neteler) [inline content/download/data.en.md]:
```suggestion
  <span class="mwl"><a href="https://grass.osgeo.org/grass-stable/manuals/ " target="_blank"> Climate </a></span>
  <a href="https://grass.osgeo.org/grass-stable/manuals/" class="inl btn btn-primary" target="_blank">Download ZIP (690MB)</a>
  <p>North Carolina location with climatic data time series (nc_climate_spm_2000_2012) - separate NC climatic data location</p>
  </li>
   <li>
  <span class="mwl"><a href="https://grass.osgeo.org/grass-stable/manuals/ " target="_blank"> Time series </a></span>
  <a href="https://grass.osgeo.org/grass-stable/manuals/" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
  <p>Extra time series of MODIS Land Surface Temperature (mapset to unzip in NC location): MODIS LST time series</p>
  </li>
   <li>
  <span class="mwl"><a href="https://grass.osgeo.org/grass-stable/manuals/ " target="_blank"> Projection </a></span>
  <a href="https://grass.osgeo.org/grass-stable/manuals/" class="inl btn btn-primary" target="_blank">Download ZIP (2MB)</a>
  <p>NAD83(HARN) / North Carolina, EPSG 3358)</p>
```

I have taken out "related tutorial" but maybe it should have been a link?

File: content/download/data.en.md [hunk 13]
```diff
@@ -1,39 +1,107 @@
...
+  </li>
+   <li>
+  <span class="mwl"><a href="https://grass.osgeo.org/uploads/grass/sampledata/slovakia3d_readme.txt" target="_blank"> 3D / voxel </a></span>
+  <a href="https://grass.osgeo.org/uploads/grass/sampledata/slovakia3d_grass7.tar.gz" class="inl btn btn-primary" target="_blank">Download ZIP (8.9MB)</a>
```
Comment (neteler) [inline content/download/data.en.md]:
```suggestion
  <span class="mwl"><a href="https://grass.osgeo.org/sampledata/slovakia3d_readme.txt" target="_blank"> 3D / voxel </a></span>
  <a href="https://grass.osgeo.org/sampledata/slovakia3d_grass7.tar.gz" class="inl btn btn-primary" target="_blank">Download ZIP (8.9MB)</a>
```

fix URLs to standard URLs

File: content/download/data.en.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ninsbl) [reply content/download/data.en.md]: I stick to "dataset" for now.

**Comment by ninsbl** [comment]
Should be an acceptable version 1.0 now. But I am happy to add more changes. On the old page there is (as probably also "Spearfish") more stuff for a museum section...
Let me know if you would expect more descriptive text...

**Comment by mlennert** [comment]
This looks really good !
One suggestion: 
"Note: an easy way to download sample data is from the download button on the GRASS GIS startup screen."
I think this could be illustrated with a screenshot like [this one.](https://trac.osgeo.org/grass/attachment/wiki/Grass7/NewFeatures74/grass740_startup_sample_data_download_arrow.png)

**Comment by ninsbl** [comment]
Thanks, motitz, very good suggestion. Will add this.

**Pull request merged**